examples/third_party_integration/flux/template/deployment.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-app
+  namespace: hello-world
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hello-app
+  template:
+    metadata:
+      labels:
+        app: hello-app
+    spec:
+      containers:
+        - name: hello-app
+          image: matrus2hub/go-hello-world@sha256:2c4bd6771dbb88db8d8294245d2937f797ff30708d0647aec564e8cd7851c2b5
```

Comment:
@cpanato  @vladimirvivien  do we have an image coming from the kubernetes-sigs that we can use here instead of this ? It would be great if there is one. I did a quick search through the `test-infra` and did not find one. May be we can just use a simple upstream nginx image instead ? Sorry for being very picky on the source of the images.

Reply:
@harshanarayana You are completely right, I switched to nginx instead.

Reply:
Along the same line of questioning, @cpanato @harshanarayana, since nginx is on Dockerhub, will repo-testing run into pull throttling issues, causing it to fail ?

Reply:
We might very well run into that. But I did a bit of digging around in the test-infra and `kubernetes-sigs` or and I found a combination of these images used already. 

Most of the nginx images used in the `k-sigs` are from direct dockerhub. But I did find a few other options 

1. registry.k8s.io/nginx-slim:0.8 (https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/1712c870e2ed773c3de0be8138502d8bb82c39d0/examples/kubernetes/multishare/maxshares/demo-ss-512.yaml#L21)
2. mcr.microsoft.com/oss/nginx/nginx:1.19.5 (https://github.com/kubernetes-sigs/blob-csi-driver/blob/9c74b4a3d14683a26958af9cd2a4f65656f5569e/deploy/example/statefulset.yaml#L20)

I believe it should be ok to use the default dockerhub images so long as we are using the same docker registry credentials in prow for this repo jobs as the one being used for others. cc @cpanato  @vladimirvivien

Reply:
@vladimirvivien  Let us get this one merged. We can revisit the images from a different registry again if we start seeing flaky tests because of the rate limiting.

Reply:
Thanks @harshanarayana for looking into this.

/lgtm
/approve